apps/user_ldap/lib/Mapping/AbstractMapping.php

```diff
@@ -147,7 +147,7 @@ public function getNamesBySearch($search) {
 			WHERE `owncloud_name` LIKE ?
 		');
 
-		$res = $query->execute(array($search));
+		$res = $query->execute(array($this->connection->escapeLikeParameter($search)));
```

Comment:
I'm not sure if I did the right thing here. Can someone confirm? Should `$search` be able to contain characters such as `_` and/or `%`?